### ISSUE TITLE
Is Smoke Detectors Supported?

### ISSUE THREAD

**Issue by malmentm** [swttt/com.swttt.homekit#169]
Is Smoke Detectors Supported?

**Comment by robertklep** [comment]
Can you name a brand and model?

**Comment by malmentm** [comment]
Do it have to be a special brand? Or can i just add any smoke detector that's supported by Homey and it will show up in homekit with your app?

**Comment by robertklep** [comment]
No, I need a specific brand and model to check if it _might_ be supported.

**Comment by malmentm** [comment]
Ok, i have not bought anyone yet. Just wondering :) Another thing that i have noticed with you Apps is that the battery status of my Aqara sensors does not show in homekit as they do with the other homekit app on the athom store. Any ideas why? The status i avaible in Homey

**Comment by robertklep** [comment]
Battery status for sensors hasn't been supported by Homey for very long, and development of Homeykit mostly stopped so it's simply not implemented yet.

**Comment by malmentm** [comment]
Ok, so there will be no new versions of Homeykit?

**Comment by robertklep** [comment]
Probably not for a while, at least not from me (I stopped using my Homey) 😬

**Comment by malmentm** [comment]
Ok :) i understand
